### PR TITLE
Magento_ProductRecommendationsStorefront - phtml and Layout file path updated.

### PR DESCRIPTION
**Pull request by rajesh-mage**
## Purpose of this pull request

This pull request (PR) Created to update Phtml and Layout file path.

## Affected DevDocs pages

https://devdocs.magento.com/recommendations/customize.html

### CODE REVIEW

**Review by BarnyShergold — APPROVED** [review]
Good update

File: src/recommendations/customize.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ ee_only: True
 
 When you install the Product Recommendations module, Magento creates the `ProductRecommendationsLayout` directory. This directory contains template files that you can customize to change how the recommendations appear on your storefront. Specifically, you can modify or override the following template:
 
-`<your theme>/ProductRecommendationsLayout/web/template/recommendations.html`
+`<your theme>/Magento/ProductRecommendationsLayout/web/template/recommendations.html`
```
Comment (ajithkumar-maragathavel) [inline src/recommendations/customize.md:9]: `<your theme>/Magento_ProductRecommendationsLayout/web/template/recommendations.html`

Comment (ajithkumar-maragathavel) [reply src/recommendations/customize.md:9]: When you are overriding the HTML file in the theme, the folder name must be Magento_ProductRecommendationsLayout.

File: src/recommendations/customize.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ When you [create a recommendation](https://docs.magento.com/user-guide/marketing
 1. Change to the `layout` directory:
 
     ```bash
-    cd `<your theme>/ProductRecommendationsLayout/layout`
+    cd `<your theme>/Magento/ProductRecommendationsLayout/layout`
```
Comment (ajithkumar-maragathavel) [inline src/recommendations/customize.md:36]: Folder name must be Magento_ProductRecommendationsLayout 
cd `<your theme>/Magento_ProductRecommendationsLayout/layout`

**Review by ajithkumar-maragathavel — CHANGES_REQUESTED** [review]
Nice finding. Please update the folder name to `Magento_ProductRecommendationsLayout`.

File: src/recommendations/customize.md (hunk 1 above)
Comment (rajesh-mage) [reply src/recommendations/customize.md]: Hi @ajithkumar-maragathavel,
Thanks for your review and update. The mentioned changes are done.

File: src/recommendations/customize.md (hunk 2 above)
Comment (rajesh-mage) [reply src/recommendations/customize.md]: Hi @ajithkumar-maragathavel,
Thanks for your review and update. The mentioned changes are done.

**Review by BarnyShergold — APPROVED** [review]

**Review by ajithkumar-maragathavel — APPROVED** [review]
Thanks !! Good one.

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8918`.

**Pull request merged**